.github/workflows/publish-flatpak-repo.yml

```
@@ -0,0 +1,380 @@
+            --pages-url "${{ env.PAGES_URL }}" \
+            --homepage "${{ env.HOMEPAGE_URL }}" \
+            --gpg-key-id "${{ steps.import-gpg.outputs.fingerprint }}"
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+
+      - name: Upload Pages artifact
+        uses: actions/upload-pages-artifact@v4
+        with:
+          path: flatpak-pages-site
+          include-hidden-files: true
```

actionlint (1.7.12)

[error] input "include-hidden-files" is not defined in action "actions/upload-pages-artifact@v4". available inputs are "name", "path", "retention-days"

(action)